src/main/java/io/strimzi/kafka/bridge/http/HttpUtils.java

```diff
@@ -17,15 +17,19 @@
     private static final Logger log = LoggerFactory.getLogger(HttpUtils.class);
 
     public static void sendResponse(RoutingContext routingContext, int statusCode, String contentType, Buffer body) {
-        routingContext.response().setStatusCode(statusCode);
-        if (body != null) {
-            log.debug("[{}] Response: body = {}", routingContext.get("request-id"), Json.decodeValue(body));
-            routingContext.response().putHeader(HttpHeaderNames.CONTENT_TYPE, contentType);
-            routingContext.response().putHeader(HttpHeaderNames.CONTENT_LENGTH, String.valueOf(body.length()));
-            routingContext.response().write(body);
+        if (!routingContext.response().closed() && !routingContext.response().ended()) {
+            routingContext.response().setStatusCode(statusCode);
+            if (body != null) {
+                log.debug("[{}] Response: body = {}", routingContext.get("request-id"), Json.decodeValue(body));
+                routingContext.response().putHeader(HttpHeaderNames.CONTENT_TYPE, contentType);
+                routingContext.response().putHeader(HttpHeaderNames.CONTENT_LENGTH, String.valueOf(body.length()));
+                routingContext.response().write(body);
+            }
+            routingContext.response().end();
+            logResponse(routingContext);
+        } else {
+            log.warn("[{}] Response: already closed/ended!");
```

Comment:
Is this really deserving of warn? It seems to me that it would not be uncommon in normal (and correct) usage of the bridge to see this situation. And there's nothing the user of the bridge about it. In which case it should be at most info.